server/afk.lua

```diff
@@ -1,9 +1,83 @@
 local QBCore = exports['qb-core']:GetCoreObject()
+local loggedInPlayers = {}
+local checkUser = {}
+local previousPos = {}
+local time = {}
+local timeMinutes = {
+    ['900'] = 'minutes',
+    ['600'] = 'minutes',
+    ['300'] = 'minutes',
+    ['150'] = 'minutes',
+    ['60'] = 'minutes',
+    ['30'] = 'seconds',
+    ['20'] = 'seconds',
+    ['10'] = 'seconds'
+}
 
-RegisterNetEvent('KickForAFK', function()
-	DropPlayer(source, 'You Have Been Kicked For Being AFK')
+local function updateCheckUser(source)
+    local permissions = QBCore.Functions.GetPermission(source)
+
+    for k in pairs(permissions) do
+        if Config.IgnoreGroupsForAFK[k] then
+            checkUser[source] = false
+            return
+        end
+    end
+
+    checkUser[source] = true
+end
+
+RegisterNetEvent('QBCore:Server:OnPlayerLoaded', function()
+    loggedInPlayers[source] = true
+    updateCheckUser(source)
 end)
 
-QBCore.Functions.CreateCallback('qb-afkkick:server:GetPermissions', function(source, cb)
-    cb(QBCore.Functions.GetPermission(source))
+AddEventHandler('QBCore:Server:OnPlayerUnload', function(source)
+    loggedInPlayers[source] = false
+    checkUser[source] = false
```

Comment:
```suggestion
    checkUser[source] = nil
```
How about making this nil for players that aren't logged in? I think it makes more sense like this.

Reply:
No, then it will turn to true in the loop

Reply:
True...